jena-arq/src/main/java/org/apache/jena/riot/lang/PipedTuplesStream.java

```diff
@@ -25,13 +25,16 @@
 import org.apache.jena.sparql.core.Quad ;
 
 /**
- * Implementation of a producer class that sends @{code Tuple<Node>}; must be connected to a {@code PipedRDFIterator<Tuple<Node>}. 
+ * Implementation of a producer class that sends @{code Tuple<Node>}; must be connected to a {@code PipedRDFIterator<Tuple<Node>}.
+ *
+ * @deprecated To be removed
  */
+@Deprecated
```

Comment:
Yet again

Reply:
All Piped* updated to refer to `AsyncParser`.